tests/application/admin/test_views.py

```diff
@@ -430,9 +432,84 @@ def test_form_has_checkbox_for_each_data_source(self, test_app_client, logged_in
         response = test_app_client.get("/admin/data-sources")
         page = BeautifulSoup(response.data.decode("utf-8"), "html.parser")
 
-        # TODO: Update to `admin.merge_data_sources`
-        form = page.find_all("form", action=url_for("admin.data_sources"))[1]
+        form = page.find("form", action=url_for("admin.merge_data_sources_post"))
         assert form
 
         assert find_input_for_label_with_text(form, "Police statistics 2019").get("value") == str(ds1.id)
         assert find_input_for_label_with_text(form, "2011 Census of England and Wales").get("value") == str(ds2.id)
+
+
+class TestMergeDataSourcesView:
+    def test_rdu_user_cannot_see_data_source_merge(self, test_app_client, logged_in_rdu_user):
+
+        response = test_app_client.get("/admin/data-sources/merge")
```

Comment:
I really don't like this any more. I'm seeing lots of hard-coded URLs and manually-built paths and it feels not great.

Can we stick to using `url_for`? Even if we have to add our own test that asserts our routes, it feels way better to not have to do this everywhere.